doc improvement: General doc fixes (#8641)
nrfconnect/sdk-nrf: doc/nrf/libraries/others/app_event_manager.rst

.. em_tracing_hooks_start

The Application Event Manager uses flexible mechanism to implement hooks when an event is submitted, before it is processed, and after its processing.

> Originally designed to implement event tracing, the tracing hooks can be used for other purposes as well.

Reviewer: ```suggestion
The tracing hooks are originally designed to implement event tracing, but you can use them for other purposes as well.
```
